### PR TITLE
[SDBELGA 118-119]Add new ingest format for ATS, DPA.

### PR DESCRIPTION
**Pull request by idsvn**
1) Add two new parser:
- ATS newsml 1.2 format.
- DPA newsml 2.0 format.
2) Add unit test.
3) add some missing variable and fix unit test for ANP and TASS

### CODE REVIEW

File: server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    def test_content(self):
+        item = self.item[0]
+        self.assertEqual(item["guid"], "urn:newsml:dpa.com:20090101:190603-99-492251:3")
+        self.assertEqual(item["uri"], "urn:newsml:dpa.com:20090101:190603-99-492251")
+        self.assertEqual(item["version"], "3")
+        self.assertEqual(item["type"], "text")
+        self.assertEqual(str(item["versioncreated"]), "2019-06-03 13:00:01+00:00")
+        self.assertEqual(item["pubstatus"], "usable")
+        self.assertEqual(item["ednote"], '\n          ')
```
Comment (ride90) [inline server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py]: Seems like ednote is not ingested properly. I see content in the xml file, but seems like `NewsMLTwoFeedParser.parse_item_meta` can not handle it properly.

File: server/belga/io/feed_parsers/belga_ats_newsml_1_2.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import logging
+import pytz
+
+from superdesk.io.registry import register_feed_parser
+from .base_belga_newsml_1_2 import BaseBelgaNewsMLOneFeedParser
+
+logger = logging.getLogger(__name__)
+
+
+class BelgaATSNewsMLOneFeedParser(BaseBelgaNewsMLOneFeedParser):
```
[the remark this answers is not in the document]
Comment (idsvn) [reply server/belga/io/feed_parsers/belga_ats_newsml_1_2.py]: Thank for your notice, we will add the property author into base belga.
ATS use `newsml` format. So it almost like `BaseBelgaNewsMLOneFeedParser` but have some different :
+) Timezone(+0200) is not UTC, So we need convert to UTC,
+) It have `2nd NewsComponent` in `main NewsComponent`, and it contain `ContainItem`(contain `body_html`)

File: server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py (hunk 1 above)
Comment (idsvn) [reply server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py]: Thanks

File: server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        self.assertEqual(item["urgency"], 4)
+        self.assertEqual(item["headline"], "Mehr als 200 Migranten in der Ägäis aufgegriffen")
+        self.assertEqual(item["slugline"], "")
+        self.assertEqual(item["language"], "de")
+        self.assertEqual(item["subject"], [{'qcode': '11017000', 'name': 'Migration', 'scheme': 'iptc_subject_code'},
+                                           {'qcode': '11011000', 'name': 'Flüchtling, Asyl',
+                                            'scheme': 'iptc_subject_code'}])
+        self.assertEqual(item["genre"], [])
+        self.assertEqual(item["authors"], [{'uri': None, 'name': 'tsafos'}])
```
[the remark this answers is not in the document]
Comment (idsvn) [reply server/tests/io/feed_parsers/belga_dpa_newsml_2_0_test.py]: We have little information about it, we have only one file DPA from Belga. So it hardly to predict it. But We think creator is role is more reasonable than name. We will change it. Thanks.

**Pull request merged**